roles/argocd/templates/argocd-repository-secret.yaml.j2

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: seldon-gitea-repo
+  namespace: argocd
+  labels:
+    argocd.argoproj.io/secret-type: repository
+stringData:
+  username: "{{ gitops_user }}"
+  password: "{{ gitops_password }}"
```

Comment:
nit: A thought that came up, `gitops_user` and `gitops_password` aren't specified in `roles/argocd/defaults/main.yaml`

I'm not sure what's best practice. Was wondering if in future it would be better to do something like `argocd_gitops_user` and `argocd_gitops_password`?

And then leave it to the user to do something like in their `seldon_deploy_values.yaml` as:
```yaml
gitops_user: {{ seldon }}
argocd_gitops_user: {{ gitops_user }}
gitea_gitops_user: {{ gitops_user }}
```

No actions required

Reply:
We can, though don't see as priority for now. If you wish you can craft a PR :sunglasses: